src/components/CourtDateFilter/CourtDateFilterOptions.tsx

```diff
@@ -43,7 +43,6 @@ const CourtDateFilterOptions: React.FC<Props> = ({ dateRange, dispatch, customDa
         <RadioButton
           name={"courtDate"}
           id={"date-range"}
-          dataAriaControls={"conditional-date-range"}
```

Comment:
Unfortunatelly if we remove this, it breaks the collapsable radio buttons(they are aways displayed):
![image](https://user-images.githubusercontent.com/22743709/220062014-5379f32a-6ff1-4f94-b8da-ab62916fe31b.png)